api-reference/beta/api/directory_delete.md

```diff
@@ -0,0 +1,54 @@
+# Delete deletedItem
+
+Permanently delete an item in [deleted items](../resources/directory.md).
```

Comment:
Also, we should warn that, once permanently deleted, there is NO opportunity to restore this object.  It's GONE.